cpu/stm32_common/periph/uart.c

```diff
@@ -154,6 +154,41 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
     return UART_OK;
 }
 
+#ifdef MODULE_PERIPH_UART_MODECFG
+int uart_mode(uart_t uart, uart_data_bits_t data_bits, uart_parity_t parity,
+              uart_stop_bits_t stop_bits)
+{
+    assert(uart < UART_NUMOF);
+
```

Comment:
don't remember to a 100% anymore: but don't we need to (temporarily) disable the UART peripheral before being able to change its mode paramters?

Reply:
Most don't but some do L0 for example, you could ifdef I suppose.